File: examples/hello.js
#!/usr/bin/env node

function greet(name) {
    return `Olá, ${name}!`;
}

function main() {
    console.log(greet("bluebel Studio"));
    console.log("=".repeat(30));
    
    // Teste de loop
    for (let i = 1; i <= 5; i++) {
        console.log(`Contagem: ${i}`);
    }
    
    // Teste de cálculo
    const result = Array.from({length: 10}, (_, i) => i + 1).reduce((a, b) => a + b);
    console.log(`\nSoma de 1 a 10: ${result}`);
}

main();

File: examples/hello.py
#!/usr/bin/env python3

def greet(name):
    return f"Olá, {name}!"

def main():
    print(greet("bluebel Studio"))
    print("=" * 30)
    
    # Teste de loop
    for i in range(1, 6):
        print(f"Contagem: {i}")
    
    # Teste de cálculo
    result = sum(range(1, 11))
    print(f"\nSoma de 1 a 10: {result}")

if __name__ == "__main__":
    main()
